app/controllers/vluchte_controller.rb

```diff
@@ -0,0 +1,42 @@
+# Entry point for the webhook resource
+class VluchteController < ApplicationController
+  before_action :validate, only: [:post]
+
+  def post
```

Comment:
Misschien een create actie -> models in de app maken -> Mails later async versturen. 

Kunnen we altijd de records in de app nog checken als iemand klaagt over zijn mail oid.

Reply:
Ben hier momenteel het meest fan gaan. Ik zal binnenkort eens kijken of ik even docker netje op kan zetten met extra worker